sdk/keyvault/azure-security-keyvault-certificates/src/certificate_serializers.cpp

```diff
@@ -381,3 +381,47 @@ std::string CertificateCreateParametersSerializer::Serialize(
 
   return parameter.dump();
 }
+
+std::string CertificateContactsSerializer::Serialize(
+    std::vector<CertificateContact> const& contacts)
+{
+  json response;
```

Comment:
response?   isn't it like the `payload` or `request` or something like that?  Why did you pick `response`?